### ISSUE TITLE
qTox version number

### ISSUE THREAD

**Issue by nweyand** [qTox/qTox#4112]
When downloading qTox from the website, the file should be named something like setup-qtox-v1.7.0 instead of only setup-qtox, as this would allow to download new versions to the same folder without overwriting the previous installer.

Also, it would allow to determine the version number of an installer at first glance, which is quite nice to have. ;)

**Comment by Hasshu** [comment]
> Also, it would allow to determine the version number of an installer at first glance, which is quite nice to have.

I second this.

**Comment by pascalberger** [comment]
This would also help to maintain the [Chocolatey Package](https://chocolatey.org/packages/qtox).

**Comment by sudden6** [comment]
@tux3 do you think you can implement this? Maybe replace setup-qtox-latest with a link to the current version.

**Comment by ghost** [comment]
Ditto for [macOS release](https://github.com/qTox/qTox/releases/tag/v1.8.1) qTox-1.8.1.dmg with qTox-1.8.1.dmg.sig file as well.

**Comment by Hasshu** [comment]
By the way, how up-to-date are the recent non-Linux builds, <i>actually</i>?

At least under Windows, the toxcore version number seems to get updated every now and then, while qTox itself is stuck at "v1.7.0".

**Comment by DjDiabolik** [comment]
By the way because here on github it's present 1.8.1 and latest for windows it's 1.7.0 from site.....

**Comment by DjDiabolik** [comment]
Now it's setup-qtox-1.8.1.exe....... but on version inside "Info about qtox" i continue to read 1.7.0... it's normal ?

Add a feature to check upgrade directly from app........... i thinks it's a good.

**Comment by Hasshu** [comment]
Looks like it just appends the version number of the [latest release](/qTox/qTox/releases/latest), instead of the build's actual one.

@tux3 Is that the expected behavior?

**Comment by tux3** [comment]
@Hasshu Yes, and those should match. What were you expecting instead?

**Comment by Hasshu** [comment]
@tux3 Hmm... They do match now. You see, I downloaded the "v1.8.1" installer soon after #4177 was merged, but it actually contained an older version, for some reason (see also what @DjDiabolik said above).

**Comment by tux3** [comment]
@Hasshu It might have been still building on the server when you downloaded it, my apologies for announcing it a bit too soon then!

**Comment by Hasshu** [comment]
@tux3 Ah, no worries. The proper build is up now, so it's fine.

**Comment by DjDiabolik** [comment]
I can confirm i have upgrade to 1.8.1 about one or two days ago.....
